include/RevExt.h

```diff
@@ -51,33 +51,25 @@ struct RevExt {
   /// RevExt: sets the internal compressed instruction table
   void SetCTable( std::vector<RevInstEntry>&& InstVect ) { ctable = std::move( InstVect ); }
 
-  /// RevExt: sets the optional table (used for variant-specific compressed encodings)
-  void SetOTable( std::vector<RevInstEntry>&& InstVect ) { otable = std::move( InstVect ); }
-
   /// RevExt: retrieve the extension name
   std::string_view GetName() const { return name; }
 
   /// RevExt: baseline execution function
   bool Execute( unsigned Inst, const RevInst& Payload, uint16_t HartID, RevRegFile* regFile );
 
   /// RevExt: retrieves the extension's instruction table
-  const std::vector<RevInstEntry>& GetInstTable() { return table; }
+  const std::vector<RevInstEntry>& GetTable() { return table; }
```

Comment:
Any particular reason for the change? Not opposed to it just curious

Reply:
Like I said, it's to match the names `SetTable` and `SetCTable`. I don't know why `Inst` should be in the middle of the names for `Get`ting, while not being in the middle of the names for `Set`ting.

This PR is functionally ready, but I need to find a toolchain which supports `Zfa`, and I need to add a few more tests, before this PR can be merged. We need to see if there is a newer toolchain which can be installed on `pennywise`. @jleidel

Reply:
what version would you like installed?

Reply:
I don't know -- just some GCC/Clang which supports `Zfa`. I don't know what version when that feature was added.

Reply:
It's been ratified -- it's in the lastest spec -- it's fairly simple to implement -- so I added it.

Reply:
I installed riscv-gcc 13.2.0 on `pennywise`